Multi zone guide refresh (#647)
kumahq/kuma-website: docs/docs/1.4.1/deployments/how-multi-zone-works.md

# About multi-zone deployments

> Kuma supports the use case where multiple service meshes run in different zones. This use case includes hybrid deployments where you run your services on Kubernetes and in Universal mode, on VMs or elsewhere. Your mesh environment can include multiple isolated service meshes (multi-tenancy), and workloads running in different regions, on different clouds, or in different datacenters. A zone can be a Kubernetes cluster, a VPC, or any other deployment you need to include in the same distributed mesh environment.

Reviewer: > Kuma supports running your service mesh in multiple zones. It is even possible to run with a mix of Kubernetes and Universal zones.
Probably shorter and simpler.

Author: While I agree it's shorter and simpler and the one above is wordy, those 2 sentences do not explain what the zone is.

Reviewer 2: Yeah, some of this belongs in a higher-level "about Kuma" topic, not here. Originally I was trying to trim and associate a bunch o' different bits of explanation from the current live doc. How about something like

```suggestion
Kuma supports the a service mesh that is distributed across multiple zones. Different zones in the same mesh can run on Kubernetes or in Universal mode, in different regions, on different cloud providers, or in different datacenters. The only thing that zones have in common is their connection to the same global control plane.
```

> Kuma supports the use case where multiple service meshes run in different zones. This use case includes hybrid deployments where you run your services on Kubernetes and in Universal mode, on VMs or elsewhere. Your mesh environment can include multiple isolated service meshes (multi-tenancy), and workloads running in different regions, on different clouds, or in different datacenters. A zone can be a Kubernetes cluster, a VPC, or any other deployment you need to include in the same distributed mesh environment.

Reviewer: I was suggesting only replacing the first 2 sentences not the whole paragraph.
```suggestion
Kuma supports running your service mesh in multiple zones. It is even possible to run with a mix of Kubernetes and Universal zones. Your mesh environment can include multiple isolated service meshes (multi-tenancy), and workloads running in different regions, on different clouds, or in different datacenters. A zone can be a Kubernetes cluster, a VPC, or any other deployment you need to include in the same distributed mesh environment.
```

## Components of a multi-zone deployment

A multi-zone deployment includes:

* The **global control plane**. Accepts connections only from zone control planes -- not from any data plane proxies. It provides configurations for the global [policies](/policies) that are applied to the data plane proxies.
* The **zone control planes**. Accept connections from data plane proxies that are started in the same underlying zone, and connect to the global control plane to fetch policies for the data plane proxies that connect to them.
* The **data plane proxies**. Connect to the zone control plane in the same zone.

> * The **zone ingress** Provides ingress to services in different zones. If a zone consumes services only, and does not provide them, the zone does not require this component.

Reviewer: I think it's missing a work in "If a zone consumes services only" Did you mean "If a zone consumers only services local to its zone"?

Author: Zone A can consumer services only from Zone A, but Zone B can also consume services from Zone A, so I'm not sure it's the same

Reviewer 2: And neither option correctly explains zone ingress, right? (Yes I know I wrote the original. It's wrong.) I think we can trim to

```suggestion
* The **zone ingress** Provides ingress to services in different zones.
```

Author: > Provides ingress to services in different zones

For me, it sounds like Ingress in zone A provides an entry point to services in zone B, C, D, which is not correct.

How about "Provides ingress to local services for different zones"

Reviewer: I like the way you reworded that!

    * Configured automatically on Kubernetes if you install directly with `kumactl`.
    * Must be explicitly enabled if you install on Kubernetes with Helm.
    * Must be installed separately in Universal mode.

## How it works

Kuma manages service connectivity -- establishing and maintaining connections across zones in the mesh -- with the zone ingress and with a DNS resolver.

The DNS resolver is embedded in each data plane proxy. It resolves each service address to a virtual IP address for all service-to-service communication.

The global control plane and the zone control planes communicate to synchronize resources such as Kuma policy configurations over Kuma Discovery Service (KDS), which is a protocol based on xDS.

<center>
<img src="/images/docs/0.6.0/distributed-diagram.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
</center>

:::tip

> A zone ingress is not an API gateway. Instead, it is specific to internal cross-zone communication within the mesh. API gateways are supported in Kuma [gateway mode](/docs/1.1.1/documentation/dps-and-data-model/#gateway) which can be deployed in addition to zone ingresses.

Reviewer: Use relative paths so that we stop getting bad versions :)
